### PR TITLE
[WIP] Add return validators

### PR DESCRIPTION
**Pull request by simahawk**
**TODO**
- [ ] fixes
- [ ] update tests
- [ ] update all overrides of all modules

CC @gurneyalex

### CODE REVIEW

**Comment by gurneyalex** [comment]
@simahawk rebased on top of 13.0

File: shopinvader/services/customer.py [hunk 1]
```diff
@@ -114,3 +159,52 @@ def _prepare_create_response(self, binding):
         response = self._assign_cart_and_get_store_cache()
         response["data"] = {"id": self.partner.id, "name": self.partner.name}
         return response
+
+    def _validator_return_create(self):
+        schema = self._schema_for_assign_cart_and_get_store_cache()
+        schema.update(
+            {
+                "data": {
```
Comment (i-vyshnevska) [inline shopinvader/services/customer.py:180]: question, cannot find documentation for this, why it is wrapped to "data" key?  for schema we usually return only dict with keys

**Comment by jcoux** [comment]
Rebased on top of 13.0

**Comment by jcoux** [comment]
Rebased on top of 13.0

**Pull request closed without merging**